src/utils/file_system_utils.cc

```diff
@@ -152,16 +152,23 @@ void FileSystemUtils::convertToLocalPath(QString*)
 
 bool FileSystemUtils::isRoot(const QString& path)
 {
-	return ( (path.size() == 1 && path.startsWith( '/') ) || (path.size() == 3 && path[0].isLetter() && path[1] == ':' ) );
+    return ( (path.size() == 1 && path.startsWith( '/') ) || (path.size() == 3 && path[0].isLetter() && path[1] == ':' ) || (path.size() == 1)  );
```

Comment:
this condition could be simplified to `return ( (path.size() == 3 && path[0].isLetter() && path[1] == ':' ) || (path.size() == 1)  );`, right?